docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -116,14 +116,13 @@ as a new set of time series. In this section you're going to configure a recordi
 offered by Grafana.
 
 1. Open [Grafana Alerting](http://localhost:9000/alerting/list).
-2. Click "New alert rule", which also allows you to configure recording rules.
+2. Click "New recording rule", which also allows you to configure recording rules.
```

Comment:
Is "New recording rule" a UI label? If so, we should use bold instead of quotes.

Reply:
After checking again, I'll push again

Reply:
I think it's better to change "Click **New recording rule**"

![image](https://github.com/grafana/mimir/assets/43164634/fbb5bf2e-5743-4306-9144-a84660f95105)